### ISSUE TITLE
how can i read data from kv-ep21v connect sensor pr-g51n with mcprotocol

### ISSUE THREAD

**Issue by NguyenDucQuan12** [plcpeople/mcprotocol#15]
I have the following connectors: kv-3000, kv-c32x, kv-ep21v connected to the pr-g51n photoelectric sensor. I want to get the signal when the sensor is blocked, however I have tried most of the ways but still cannot read the signal, does this library work with Keyence plc?
![image](https://github.com/user-attachments/assets/364904fd-155f-4365-8c25-d6f18a255a24)
![image](https://github.com/user-attachments/assets/f1810b16-23de-43e7-8349-ca704e644028)

**Comment by plcpeople** [comment]
I don't think it will work with a Keyence PLC - I have never used one but I can't find any information suggesting a Keyence PLC talks using the exact same protocol as a Mitsubishi.  You might have better luck getting data out of it using Modbus/TCP, using the "mbtcpprotocol" library of ours or many other Modbus implementations in JS or other language.  Can anyone comment who knows more about Keyence?
